.github/workflows/ci.yaml

```diff
@@ -0,0 +1,93 @@
+# This config uses industrial_ci
+# (https://github.com/ros-industrial/industrial_ci.git).  For
+# troubleshooting, see readme
+# (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+      - "*-devel"
+      - "**"
```

Comment:
You can remove build all on branches, since it will still build on all PRs

Reply:
Done.